GeneratorInterface/GenFilters/interface/LHEGenericFilter.h

```diff
@@ -49,8 +49,10 @@ class LHEGenericFilter : public edm::EDFilter {
   
   edm::EDGetTokenT<LHEEventProduct> src_;
   int numRequired_;                // number of particles required to pass filter
-  bool acceptMore_;                // if true (default), accept numRequired or more.
-                                   // if false, accept events with exactly equal to numRequired.
+  std::string acceptLogic_;        // LT  meaning <
```

Comment:
hi, I am not sure I understand what/how to do it. 
to reduce the number of string comparisons to 1 is easy, but what do you mean with "always valid"?